datafusion/optimizer/src/common_subexpr_eliminate.rs

```diff
@@ -84,11 +84,11 @@ type IdArray = Vec<(usize, Identifier)>;
 /// - The number of occurrences and
 /// - The DataType
 /// of an expression.
-type ExprStats = HashMap<Identifier, (usize, DataType)>;
+type ExprStats = HashMap<Identifier, usize>;
 
 /// A map that contains the common expressions extracted during the second, rewriting
 /// traversal.
```

Comment:
perhaps we can also update the docstring here to mention that the map contains the alias

```suggestion
/// A map that contains the common expressions and their alias extracted during the second, rewriting
/// traversal.
```

Reply:
Thanks! Fixed in https://github.com/apache/datafusion/pull/10939/commits/be2e5606cbe34b9d6c0b3d8cad56499a7c5ef96e together with `ExprStats` docstring.